src/p2pool/P2PoolManager.cpp

```diff
@@ -206,11 +206,13 @@ bool P2PoolManager::start(const QString &flags, const QString &address, const QS
 void P2PoolManager::exit()
 {
     qDebug("P2PoolManager: exit()");
+    if (started) {
     #ifdef Q_OS_WIN
         QProcess::execute("taskkill",  {"/F", "/IM", "p2pool.exe"});
     #else
         QProcess::execute("pkill", {"p2pool"});
     #endif
+    }
     started = false;
     QString dirName = QApplication::applicationDirPath() + "/stats/";
     QDir dir(dirName);
```

Comment:
I think the entire method can be encompassed within the if.
```suggestion
    if (started) {
        #ifdef Q_OS_WIN
            QProcess::execute("taskkill",  {"/F", "/IM", "p2pool.exe"});
        #else
            QProcess::execute("pkill", {"p2pool"});
        #endif
        started = false;
        QString dirName = QApplication::applicationDirPath() + "/stats/";
        QDir dir(dirName);
    }
```